torch/_dynamo/config.py

```diff
@@ -289,6 +289,7 @@ def _get_optimize_ddp_mode():
 
 # Whether to skip guarding on FSDP-managed modules
 skip_fsdp_guards = True
+skip_fsdp_hooks = False
```

Comment:
would be great to have comment on this describing what this config controls

Reply:
added comment